### PR TITLE
swift-inspect: clean up some of the code in `WindowsRemoteProcess`

### PR DESCRIPTION
**Pull request by compnerd**
Add some vertical whitespace to the code ejection process.  Alter the logic to clean up the memory allocation first, ignoring the error as the subsequent run will perform a new allocation and this will leak a fixed amount of memory without interrupting the process or use of the tool. No longer check the exit code of the thread as that is always guaranteed to be 0 as the module unloading path does not report any error code in the injected code (DLL).

Use the opportunity to do some simple renaming to improve the readability and create an overload for avoiding unnecessary ceremony around use of a shared constant.

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

File: tools/swift-inspect/Sources/swift-inspect/WindowsRemoteProcess.swift [hunk 1]
```diff
@@ -435,43 +437,33 @@ internal final class WindowsRemoteProcess: RemoteProcess {
...
       print("CreateRemoteThread for unload failed \(GetLastError())")
       return false
     }
-    defer { CloseHandle(hUnloadThread) }
-    let unload_wait = WaitForSingleObject(hUnloadThread, DWORD(WAIT_TIMEOUT_MS))
-    if unload_wait != WAIT_OBJECT_0 {
-      print("WaitForSingleObject on FreeLibrary failed \(unload_wait)")
-      return false
-    }
-    var unloadExitCode: DWORD = 0
```
[the remark this answers is not in the document]
Comment (compnerd) [reply tools/swift-inspect/Sources/swift-inspect/WindowsRemoteProcess.swift:468]: Done

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please build toolchain Windows platform

**Review by hjyamauchi — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test Windows platform

**Pull request merged**